scripts/patches/debug_ce.sh

```diff
@@ -6,5 +6,6 @@ source scripts/property_utils.sh
 
 SQ_HOME=$1
 
-echo "enabling debug on compute engine, listening on port 5005"
-set_property sonar.ce.javaAdditionalOpts -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005 "$SQ_HOME/conf/sonar.properties"
+port=5005
```

Comment:
in my patches, I use 5006 for ce and 5005 for web.
Probably because web always starts before ce (and existing even before ce existed).

Reply:
I switched it ;-)